docs/TASKS.md

```diff
@@ -3,6 +3,27 @@ Tasks
 
 A task describes the how, what, and when to do for a Snap job.  A task is described in a task _manifest_, which can be either JSON or YAML<sup>1</sup>.
 
+A task can be in the following states:
+- **started:** a running task
+- **stopped:** a task that is not running
+- **disabled:** a task in a state not allowed to start
```

Comment:
A task is disabled on error (by snap) and is not resumable directly. It can be re-enabled which moves it from disabled-> stopped and then is able to be started, iirc. I think phrasing something about it being disabled on error and then requiring the task to be explicitly re-enabled before being started would be good. 

Do we actually have a started state? Looking at the code [here](https://github.com/intelsdi-x/snap/blob/master/core/task.go#L50-L55) it seems like that would be called running. I think this is what would be reported by the API as the state of the task. Would it make sense to stick to those labels then?

The ones I think we'd want to talk about are: 
* running 
* disabled
* stopped
* ended